src/Vehicle/VehicleBatteryFactGroup.cc

```diff
@@ -28,6 +28,9 @@ const char* VehicleBatteryFactGroup::_chargeStateFactName           = "chargeSta
 
 const char* VehicleBatteryFactGroup::_settingsGroup =                       "Vehicle.battery";
 
+
+QMap<uint8_t, QPair<double, double>> VehicleBatteryFactGroup::_last_known_consumed{};
```

Comment:
```suggestion
QMap<uint8_t, QPair<double, double>> VehicleBatteryFactGroup::_lastKnownConsumed{};
```

Reply:
This is shared between all vehicles, so if the vehicles batteries have the same id, they will overwrite eachother